backend/helper_zero/views/search.py

```diff
@@ -19,59 +19,71 @@ def list(self, request):
         lon = request.query_params.get('lon')
         radius = request.query_params.get('radius', 10)
         zipcode = request.query_params.get('zipcode')
+        offset = int(request.query_params.get('offset', 0))
```

Comment:
What alternative besides offset=0 were you thinking the frontend passes in? It's not super clear to me what the behavior of a nonzero offset is

Reply:
assuming that the frontend makes another /search request when a user clicks "view more results", it'll increment offset and send it over so the backend can paginate the results shown

Reply:
ah i see, so it pulls the list of all results again and then grabs the page given the offset? Can you add validation here that offset >= 0 & error if it's false - then can get rid of the check for `page_index > 1` in the pagination function. Think that makes it a little clearer to understand.

Reply:
yeah each page will be a separate search request

i check page_index > 1 to check if we're past the first page of results; if so, i use the previous page index to call the built in django pagination `.has_next()` function https://docs.djangoproject.com/en/3.0/topics/pagination/

Reply:
ah i see what you mean. sounds good then!

Reply:
added a descriptive comment since it's confusing but i couldn't find a better way thus far (y)